adoptopenjdk-api-v3-models/src/main/kotlin/net/adoptopenjdk/api/v3/models/BinaryAssetView.kt

```diff
@@ -11,15 +11,20 @@ class BinaryAssetView {
     @Schema(example = "jdk8u162-b12_openj9-0.8.0")
     val release_name: String
 
+    @Schema(implementation = Vendor::class)
```

Comment:
One final thing, now that we filter on vendor, vendor should be implied by the request made, so I dont think it is necessary to add this to the BinaryAssetView as it would be duplicated data on all times

Reply:
By removing all references to vendor in that view, would we not be removing the functionality that was introduced? If we remove vendor from here, it doesn't look like it would be displayed in the frontend (although I haven't tested it as by removing that reference, I break everything else in this PR)